### PR TITLE
SFTT backend docs

### PR DESCRIPTION
**Pull request by florisdobber**
Added a new page for the new back end with the goal of creating better maintained and easier to read back end documentation. The navigation works properly and it includes the new routers and routes.

### CODE REVIEW

File: docs/sftt/sftt-api-spec-new.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
+# SFTT API Specification
+This is the official API specification for the SFTT back end. The back end is implemented as is described in the documentation below and should correspond exactly with how the back end behaves. If you find any inconsistencies please create a PR to amend the error.
+
+!!! info "Java Web Tokens (JWT)"
```
Comment (jackblanc) [inline docs/sftt/sftt-api-spec-new.md:4]: JSON Web Tokens, not Java

File: mkdocs.yml [hunk 2]
```diff
@@ -11,6 +11,8 @@ markdown_extensions:
   - admonition
   - pymdownx.details
   - pymdownx.superfences
+  - toc:
```
Comment (jackblanc) [inline mkdocs.yml:14]: ?

**Review by jackblanc — APPROVED** [review]
Looks good

File: mkdocs.yml (hunk 2 above)
Comment (florisdobber) [reply mkdocs.yml:14]: This is to limit the table of contents to a depth 3, so only #header1, ##header2 and ###header3 show up on the navigation.

**Pull request merged**